test/integration/util/util.go

```diff
@@ -50,6 +52,14 @@ func (m *MinikubeRunner) RunCommand(command string, checkError bool) string {
 	return string(stdout)
 }
 
+func (m *MinikubeRunner) EnsureRunning() error {
+	if m.GetStatus() != "Running" {
+		m.RunCommand("start", true)
```

Comment:
Ah you're right. I got it confused with kubectl runner. This is fine for now.

But then you should drop the "error" return value from this function since you just return nil.